develop/GMapHeatmap.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable */
+import React, { PropTypes } from 'react';
+import compose from 'recompose/compose';
+import defaultProps from 'recompose/defaultProps';
+import withStateSelector from './utils/withStateSelector';
+import withHandlers from 'recompose/withHandlers';
+import withState from 'recompose/withState';
+import withContext from 'recompose/withContext';
+import withProps from 'recompose/withProps';
+import withPropsOnChange from 'recompose/withPropsOnChange';
+import ptInBounds from './utils/ptInBounds';
+import GoogleMapReact from '../src';
+import SimpleMarker from './markers/SimpleMarker';
+import { createSelector } from 'reselect';
+import { susolvkaCoords, generateMarkers, heatmapData } from './data/fakeData';
```

Comment:
Lets put first _global imports_ and then _local imports_, meaning that we should have something like: 
```
import React, { PropTypes } from 'react';
import { blablabla } from 'recompose';
import { createSelector } from 'reselect';

import GoogleMapReact from '../src';
import SimpleMarker from './markers/SimpleMarker';
import withStateSelector from './utils/withStateSelector';
import ptInBounds from './utils/ptInBounds';
import { susolvkaCoords, generateMarkers, heatmapData } from './data/fakeData';
```